### ISSUE TITLE
2FA

### ISSUE THREAD

**Issue by that-mike-bal** [Automattic/wp-calypso#63823]
#### Steps to reproduce

1. Starting at URL:
2.
3.
4.

**Comment by arunsathiya** [comment]
@that-mike-bal 👋 I spotted several issues without any description: https://github.com/Automattic/wp-calypso/issues/created_by/that-mike-bal 

May I ask if these are placeholders for something you'd be something very soon, or can they be closed?

**Comment by arunsathiya** [comment]
- Looks like these are expected - https://github.com/Automattic/wp-calypso/issues/64026#issuecomment-1137332946

**Comment by that-mike-bal** [comment]
Yes @arunsathiya - we are working with a tool called Zenhub that integrates with Github and will be adding epics and labels to make it easy to organize  projects and initiatives at a high level. They will start empty but the plan is for leads to include a high level description of the project and links to relevant p2 posts.
